### ISSUE TITLE
xlrd xlsxwriter modules cannot be found

### ISSUE THREAD

**Issue by 1500cc** [odoo/docker#206]
ODOO is downloaded unmodified from docker hub
"ODOO_VERSION": 11.0
"ODOO_RELEASE": 20180808

**When trying to install the odoo app:**
[link to report xlsx](https://apps.odoo.com/apps/modules/11.0/report_xlsx/)

**I get error:**
Unable to install module "report_xlsx" because an external dependency is not met: No module named xlrd

I noted they are included in the base_requirements.txt in the later commits of odoo as mentioned in https://github.com/odoo/odoo/pull/14330

Could we include these changes into the docker image please.
(current workaround is to include the xlrd and xlsxwriter from the downloaded python packages to the odoo addons folder and adding in the manifest manually)

**Comment by trygvis** [comment]
This is supposedly [already fixed](https://github.com/odoo/odoo/pull/14330#issuecomment-381130109), but I have the same problem with `12.0` from docker.

**Comment by PhatDang** [comment]
Hey, me too. I facing the same issue. How can fix? I use Odoo 12.

**Comment by trygvis** [comment]
@PhatDang I created a custom image that is based on this image.

My Dockerfile:

```
FROM odoo:12

RUN pip3 install phonenumbers
VOLUME ["/var/lib/odoo", "/mnt/extra-addons"]

EXPOSE 8069 8072
USER odoo
ENTRYPOINT ["/entrypoint.sh"]
CMD ["odoo"]
```

**Comment by d-fence** [comment]
Fixed here: 6011cc7931c36b5e3bfb2caca17749e005146f73
Thanks
